src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/KafkaWebJobsBuilderExtensions.cs

```diff
@@ -40,5 +41,22 @@ public static IWebJobsBuilder AddKafka(this IWebJobsBuilder builder, Action<Kafk
 
             return builder;
         }
+
+        internal static IWebJobsBuilder AddKafkaScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
+        {
+            IServiceProvider serviceProvider = null;
+            var scalerProvider = new Lazy<KafkaScalerProvider>(() => new KafkaScalerProvider(serviceProvider, triggerMetadata));
+            builder.Services.AddSingleton((Func<IServiceProvider, IScaleMonitorProvider>)delegate (IServiceProvider resolvedServiceProvider)
```

Comment:
Looks like we don't want IScaleMonitorProvider, and we only want to use target based scaling going forward. Can we remove the support for this. cc @alrod Should we throw not implemented exception for iscalemonitor or what is the recommended approach?